content/contact/index.rst

```diff
@@ -21,6 +21,20 @@ social:
     text: 'birds@oddbird.net'
     url: 'mailto:birds@oddbird.net'
     after: 'by Email'
+cta_footer:
+  - title: 'Start the conversation…'
+    susytitle: 'Partner with OddBird on Your Next Project'
+    contact: 'miriam'
+    action:
+      text: 'Schedule a call with %s'
+      url: '/contact/'
+    content: |
+      **Interested in a chat with %s to find out more?**
+      Ask us anything;
+      We're happy to answer your questions,
+      big or small.
+      If your business is a good fit,
+      we'll offer a free in-depth project assessment!
```

Comment:
What does this "in-depth project assessment" entail?

Reply:
Basically, we're talking about doing the "overview/goals" document on a free phone call with the client.